### Hi there ðŸ‘‹

- ðŸ’¬ [My GitHub Pages](https://jingwora.github.io/index.html)

Bioinformatics (Under productionðŸŒ±):

- ðŸ“• [bioinformatics-tools](https://github.com/jingwora/bioinformatics-on-cloud)

| Tools | Brief Description | Written in | Link |
|:-----------:|:---------:|:----------:|:-----------:|
| SAMtools    | A   suite of tools for manipulating alignments in the SAM format                                              | C          | [github](https://github.com/samtools/samtools) [demo](https://github.com/jingwora/bioinformatics-tools/tree/main/tools/linux-command) |
| Pysam       | A   Python module for reading and manipulating SAM/BAM files                                                  | Python     | [github](https://github.com/pysam-developers/pysam) [demo](https://github.com/jingwora/bioinformatics-on-cloud/blob/main/tools/pysam/pysam_v-01.ipynb) |
| GATK       | The Genome Analysis Toolkit, a software package for variant discovery in high-throughput sequencing data | Java     | [github](https://gatk.broadinstitute.org/) |
| Picard       | A set of Java command line tools for manipulating high-throughput sequencing (HTS) data and formats such as SAM/BAM/CRAM and VCF | Java     | [github](https://broadinstitute.github.io/picard/) |
| tsinfer     | A   tool for inferring true evolutionary trees from large multiple sequence   alignments                      | C++        | [github](https://github.com/tskit-dev/tsinfer) [demo](https://github.com/jingwora/bioinformatics-on-cloud/blob/main/tools/tsinfer/Tskit-tsinfer_v-01.ipynb) |
| msprime     | A   coalescent simulation program that implements the forward-time population   genetics simulation algorithm | Python     | [github](https://github.com/tskit-dev/msprime) [demo](https://github.com/jingwora/bioinformatics-on-cloud/blob/main/tools/msprime/msprime_v-01.ipynb) |
| BWA         | A   software package for mapping low-divergent sequences against a large   reference genome                   | C          | [github](https://github.com/lh3/bwa) [demo](https://github.com/jingwora/bioinformatics-on-cloud/blob/main/tools/bwa/bwa_v-02-03.ipynb) |
| FastQC      | A   quality control tool for high throughput sequence data                                                    | Java       | [github](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/) [demo](https://github.com/jingwora/bioinformatics-tools/blob/main/tools/fastqc/fastqc.ipynb) |
| Trimmomatic | A   flexible read trimming tool for Illumina NGS data                                                         | Java       | [github](https://github.com/timflutre/trimmomatic) [demo](https://github.com/jingwora/bioinformatics-tools/blob/main/tools/trimmomatic/trimmomatic.ipynb) |
| InSilicoSeq | A tool for simulating NGS reads from genomes and transcriptomes                                                         | Python       | [github](https://github.com/ScrippsGenomeDynamics/InSilicoSeq) |
 
- ðŸ“• Project Management
  - ðŸ“„ [bioconda mumba](https://github.com/jingwora/bioinformatics-tools/blob/main/tools/bioconda/bioconda.ipynb)
  - ðŸ“„ [Makefiles](https://github.com/jingwora/bioinformatics-tools/blob/main/tools/Makefile/Makefile-tutorial.ipynb)
  - ðŸ“„ [Snakemake](https://github.com/jingwora/bioinformatics-on-cloud/blob/main/tools/Snakemake/snakemake-tutorial/example-workflow.ipynb)
  - ðŸ“„ [Docker](https://github.com/jingwora/bioinformatics-tools/blob/main/tools/docker/docker-tutorial.ipynb)
  - ðŸ“„ [Singularity](https://github.com/jingwora/bioinformatics-tools/blob/main/tools/singularity/Singularity-01-lolcow.ipynb)
 
- ðŸ“• [C tutorial](https://github.com/jingwora/c_tutorial)


<!--
**jingwora/jingwora** is a âœ¨ _special_ âœ¨ repository because its `README.md` (this file) appears on your GitHub profile.

Here are some ideas to get you started:

- ðŸ”­ Iâ€™m currently working on ...
- ðŸŒ± Iâ€™m currently learning ...
- ðŸ‘¯ Iâ€™m looking to collaborate on ...
- ðŸ¤” Iâ€™m looking for help with ...
- ðŸ’¬ Ask me about ...
- ðŸ“« How to reach me: ...
- ðŸ˜„ Pronouns: ...
- âš¡ Fun fact: ...
-->
